src/test/java/alison/DeadlineTest.java

```diff
@@ -0,0 +1,18 @@
+package alison;
+
+import alison.exception.AlisonException;
+import alison.task.Deadline;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
I like the lexicographic way in which you arrange your import statements in all your classes, perhaps a line break between lines 4 and 5 and lines 5 and 6 would make the import statements look neater? 🤔